gnomad_constraint/utils/constraint_basics.py

```diff
@@ -0,0 +1,83 @@
+# noqa: D100
+
+import hail as hl
+
+from gnomad.utils.constraint import (
+    trimer_from_heptamer,
+    annotate_mutation_type,
+    collapse_strand,
+)
+from gnomad.utils.vep import add_most_severe_csq_to_tc_within_vep_root
+
+
+def add_vep_context_annotations(
+    ht: hl.Table, annotated_context_ht_path: str
+) -> hl.Table:
+    """
+    Add annotations from VEP context Table to gnomAD data.
+
+    Function adds the following annotations:
+        - context
+        - methylation
+        - coverage
+        - gerp
+        - pass_filters
+
+    Function drops `a_index`, `was_split`, and`colocated_variants` annotations from gnomAD data.
+
+    .. note::
+        Function expects that multiallelic variants in the VEP context Table have been split.
+
+    :param ht: gnomAD exomes or genomes public Hail Table.
+    :param annotated_context_ht_path: Path to VEP context Table.
+    """
+    context_ht = hl.read_table(annotated_context_ht_path).drop("a_index", "was_split")
+    context_ht = context_ht.annotate(vep=context_ht.vep.drop("colocated_variants"))
+    return ht.annotate(**context_ht[ht.key])
+
+
+def prepare_ht_for_constraint_calculations(ht: hl.Table):
```

Comment:
I deleted the parameter `annotate_coverage`, because under any circumstance in the pipeline the coverage will be annotated.